keps/sig-node/4210-max-image-gc-age/README.md

```diff
@@ -208,12 +208,13 @@ Additional tests will be added to this file to cover the garbage collection e2e.
 
 - Configuration field added to the Kubelet (disabled by default)
 - Feature supported by Kubelet Image Manager
-- Unit tests and e2e tests added
+- Unit tests
 - Add a metric `kubelet_image_garbage_collected_total` which tracks the number of images the kubelet is GC'ing through any mechanism.
 
 #### Beta
```

Comment:
since PRR relies on the metric `kubelet_image_garbage_collected_total`, can we add as a goal to make sure the metric is documented? (I didn't find it quickly: https://kubernetes.io/search/?q=%22image_garbage_collected_total%22, sorry if I missed something)



https://kubernetes.io/search/?q=kubelet_image_garbage_collected_total

Reply:
I have added a goal for beta to better document it